app/controllers/admin/schools/induction_coordinators_controller.rb

```diff
@@ -49,20 +45,11 @@ def set_school
     end
 
     def school_induction_tutor_attributes
-      user = @school.induction_tutor
-      if user
-        {
-          user_id: user.id,
-          email: user.email,
-          full_name: user.full_name,
-        }
-      else
-        {}
-      end
-    end
-
-    def school_using_this_email(email)
-      Identity.find_user_by(email:).schools.first
+      {
+        induction_tutor: @school.induction_tutor,
+        email: @school.induction_tutor&.email,
+        full_name: @school.induction_tutor&.full_name,
```

Comment:
Could these use the attr_reader for `school` now?

Reply:
I can do that but for consistency I also should change the rest of ocurrencies in the controller.

Reply:
Done